app/models/user.rb

```diff
@@ -16,8 +16,11 @@ class User < ApplicationRecord
 
   has_many :friend_requests_sent, -> { where(status: 'pending') },
            class_name: 'Friendship', foreign_key: 'user_id'
+  has_many :pending_friends, through: :friend_requests_sent, source: :user
   has_many :friend_requests_received, -> { where(status: 'pending') },
            class_name: 'Friendship', foreign_key: 'friend_id'
+  has_many :friend_requests, through: :friend_requests_received, source: :user
+
   def friend?(user)
     friendship = Friendship.find_by(user_id: user.id, friend_id: id, status: 'confirmed') ||
                  Friendship.find_by(user_id: id, friend_id: user.id, status: 'confirmed')
```

Comment:
Add tests for the above associations and methods. Also, add test for the association on line 11. If the methods are not being used outside the class, declare them as private and don't write tests for them.